test/functional/lib/tests.cpp

```diff
@@ -178,7 +212,35 @@ MpResetMtu(
 
 using unique_fnmp_mtu_handle = wil::unique_any<const TestInterface *, decltype(::MpResetMtu), ::MpResetMtu>;
 
+#endif
+
+#ifdef KERNEL_MODE
+#define POOLTAG_TEST 'tsTX' // XTst
+
+_IRQL_requires_max_(DISPATCH_LEVEL)
+XDP_STATUS
+XdpFuncXskNotifyCallback(
```

Comment:
how is this tested if it just always returns STATUS_SUCCESS?

Reply:
I don't get the question. For me I thought this is something dummy function.
Any idea @nigriMSFT ?

Reply:
Michael means we need a test for if/when this function returns a failure.

Reply:
Right, and it also needs to verify the parameters provided by XDP itself are valid. In other words, this function needs to validate its inputs and be able to return success or failure, depending on the test case.

That said, I didn't realize many of the test cases have not been implemented in kernel mode yet, so this being a stub for now makes sense.

Reply:
There is 1 success value and 4 error values in XDP_STATUS. Should I prepare each callback for each value?
If the the `Result` input parameter can be controlled from test code, we can switch the return value though.
Where is the callback called? As far as I checked it seems not being called from xdp code. nmr?

Reply:
Right I was only trying to get things to compile at this point, hence, its a stub right now